api/notd/token_metadata_processor.py

```diff
@@ -4,10 +4,13 @@
 from core.exceptions import BadRequestException
 from core.exceptions import NotFoundException
 from core.requester import Requester
+from core.s3_manager import S3Manager
+from core.util import date_util
 from core.web3.eth_client import EthClientInterface
 
 from notd.model import RetrievedTokenMetadata
 
+_BUCKET = 's3://kibatest'
```

Comment:
pass this in as a parameter rather than defining globally and make it an environment variable read in worker / application / local_testing

Reply:
I don't get the second part of passing it into the worker or application

Reply:
so in worker / application / local_testing you should have something like `TokenMetadataProcessor(..., bucketName=os.environ['S3_BUCKET_NAME'])`
and then use bucketName inside TokenMetadataProcessor